project/service/src/main/java/vasilizas/myservice/person/MyService.java

```diff
@@ -1,18 +1,18 @@
-package service.facade;
+package vasilizas.myservice.person;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import static factory.Factory.createStudent;
-import static factory.Factory.createTeacher;
-import static repository.StudentRepository.studentList;
-import static repository.TeacherRepository.teacherList;
+import static vasilizas.factory.Factory.createStudent;
+import static vasilizas.factory.Factory.createTeacher;
+import static vasilizas.repository.StudentRepository.studentList;
+import static vasilizas.repository.TeacherRepository.teacherList;
 
-public class Service {
+public class MyService {
 
     public static final Logger log = LoggerFactory.getLogger("Service");
 
-    private Service() {
+    private MyService() {
     }
```

Comment:
Но вообще-то сервисы обычно не утилиты, а синглтоны. 

```suggestion
    private MyService() {
        // blank default constructor for utility class
    }
```

Reply:
исправил. но вроде бы сонар ругается если конструктор публичный, на приватный он не ругается